src/ConsulResponse.php

```diff
@@ -30,8 +34,13 @@ public function getStatusCode(): int
         return $this->status;
     }
 
-    public function json(): ?array
```

Comment:
Why did you remove the typehint?

Reply:
Because put key request returns true

https://www.consul.io/api-docs/kv#sample-requests

Reply:
oh, thanks